2129094 stf cleanup (#393)
infrawatch/documentation: README.md

    brew install asciidoctor make

> Alternatively, to make the "make" utility available on PATH, you can install the XCode development environment.

Reviewer: Recommend:  'Alternatively, you can install the XCode development environment so that the "make" utility is available on PATH.
Just to avoid the 'make the make utility' phrasing, although I can see this wasn't a part of the commit. It's up to you.

> Alternatively, to make the "make" utility available on PATH, you can install the XCode development environment.

Author: ```suggestion
Alternatively, you can install the XCode development environment so that the "make" utility is available on PATH.
```

    gem install asciidoctor asciidoctor-pdf --pre

Run `make` or `make html`, which builds HTML output. Generating
PDF output is slow, therefore use the `make pdf` command separately. To

> make both formats in one command, use `make html pdf`. To build downstream

Author: ```suggestion
make both formats in one command, use `make html pdf`. To build a downstream
```

> version perform `make BUILD=downstream browser`.

Author: ```suggestion
version, run the `make BUILD=downstream browser` command.
```

Few additional `make` targets are available on the guide level. To build

> HTML version and to view it in a new tab in a browser, enter the following commands:

Author: ```suggestion
a HTML version and to view it in a new tab in a browser, enter the following commands:
```

    cd doc-Service-Telemetry-Framework
    make browser

> To build and open PDF version, enter the following command:

Reviewer: Several instances of "open X version" instead of "open an X version" or "open the X version of..." It looks like an article is missing.

> To build and open PDF version, enter the following command:

Author: ```suggestion
To build and open a PDF version, enter the following command:
```

	# ls {VariableName}
	----

Hide or show specific blocks, paragraphs, warnings or chapters by using a special

> variable called build. You can set the value of build to either "downstream" or

Reviewer: ```suggestion
variable called `build`. You can set the value of build to either "downstream" or
```
I'm sure the standard is different for upstream, but thought I might tag it just in case.
